package.json

```diff
@@ -24,9 +25,13 @@
     "eslint-plugin-import": "2.18.2",
     "eslint-plugin-prettier": "3.1.1",
     "jest": "24.8.0",
+    "jest-date-mock": "^1.0.7",
```

Comment:
Unpinned dependencies could lead malicious code to be injected.
For more info, please see [this guide](https://docs.renovatebot.com/dependency-pinning/).